docs/source/cli_help/github.rst

```diff
@@ -26,16 +26,18 @@ a GitHub API token with ``repo`` scope is required, see https://github.com/setti
   Any existing annotations are overwritten for the branches that have an opened PR; annotations for the other branches remain untouched.
   Equivalent to ``git machete anno --sync-github-prs``.
 
-``checkout-prs``:
+``checkout-prs [--all | --by=<github-login> | --my | <PR-number-1> ... <PR-number-N>]``:
 
-  Check out the head branch of the given pull request (specified by number),
+  Check out the head branch of the given pull requests (specified by number),
   also traverse chain of pull requests upwards, adding branches one by one to git-machete and check them out locally.
   Once pull request is checked out locally, annotate local branches with corresponding pull request numbers.
+  If only one PR is given, then switch local branch to its HEAD.
```

Comment:
`HEAD` is just a single reference specific to the local repository, stored under `.git/HEAD`! See https://slides.com/plipski/git-internals/#/42

Reply:
I'd say:
```suggestion
  If only one PR is given, then switch the local repository's HEAD to its head branch.
```

Reply:
`head branch of a PR` != `HEAD of a local repository` :fuggg: